scripts/configure_jenkins.sh

```diff
@@ -150,6 +150,7 @@ jenkins-jobs --conf $CONFIG update $SCRIPTS_DIR/../jobs/integr8ly/ocp3/tower/tow
 generate_inline_script_job $SCRIPTS_DIR/../jobs/integr8ly/ocp4/install/openshift4-rhmi-install.yaml
 generate_inline_script_job $SCRIPTS_DIR/../jobs/integr8ly/ocp4/nightly/executor/osd-ocp4-testing-executor.yaml
 generate_inline_script_job $SCRIPTS_DIR/../jobs/integr8ly/ocp4/nightly/trigger/osd-ocp4-testing-nightly-trigger.yaml
+generate_inline_script_job $SCRIPTS_DIR/../jobs/integr8ly/ocp4/utils/htpasswd/Jenkinsfile
```

Comment:
Is this correct? I don't think so

Reply:
Only noticed it now. Creating a fix for that now